Ports/jfduke3d/package.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='jfduke3d'
+version='41cd46bc00633e7457d07d88c8add9f99a7d9d41'
+workdir="jfduke3d-${version}"
+launcher_name='Duke Nukem 3D'
+launcher_category='Games'
+launcher_command='duke3d'
+auth_type='sha256'
+depends=(
+  "SDL2"
```

Comment:
1) Use single quotes for non-interpolated strings
2) Use four-space indents

Both apply to `makeopts` below as well

Reply:
Done!